common/lib/xmodule/xmodule/modulestore/tests/test_mongo.py

```diff
@@ -4,6 +4,7 @@
 # pylint: disable=no-member
 # pylint: disable=protected-access
 # pylint: disable=no-name-in-module
+# pylint: disable=bad-continuation
```

Comment:
It was strange pylint check on my localdev was passing, but build on Jenkins wasn't. I don't know if there is a pylint version skew or not. That said, I couldn't figure out what the problem was about (around line 213 of test_mongo.py - and similar block right below it)